examples/preprocessors__grep/README.md

```diff
@@ -0,0 +1,69 @@
+# Selecting tests using Mocha-like "fgrep" and "grep"
+
+This example uses [cypress-select-tests](https://github.com/bahmutov/cypress-select-tests) preprocessor plugin to filter specs and tests. It is similar to how [Mocha](https://mochajs.org/) has `--fgrep` and `--grep` CLI arguments. This project provides imitation using strings.
+
+Because Cypress ignores unknown CLI parameters, you need to pass `fgrep` and `grep` arguments as environment variables, for example by using `--env` CLI argument.
+
+## Examples
+
+### fgrep to pick specs
+
+```bash
+# runs only tests from specs with "foo" in their filename
+$ npm run cypress:open -- --env fgrep=foo
```

Comment:
why though? We already have a `--spec` flag. Not only that, but we **also have a `testFiles` and `ignoreTestFiles` [configuration](https://on.cypress.io/configuration#Global)**. People are already confused about how these 2 play together. What happens when the `fgrep` and `spec` flags conflict? How does it all play together with the `testFiles` and `ignoreTestFiles` config?

Reply:
I have removed "fgrep" from the example - leaving just `grep` to avoid confusion.